internal/tls/cert/tls_cert_validator.go

```diff
@@ -27,15 +28,19 @@ func (tcv *TLSCertValidator) ValidateCertificate(certPEM []byte, keyPEM []byte)
 		Validity:        time.Now().Add(time.Hour * 24 * 365),
 	}
 
+	// Make a best effort replacement of linebreaks in cert/key if present.
+	certReplaced := []byte(strings.ReplaceAll(string(certPEM), "\\n", "\n"))
```

Comment:
Maybe rename to sanitizedCert / sanitizedKey?

Also use [ReplaceAll](https://pkg.go.dev/bytes#ReplaceAll) from the bytes package instead of converting twice.

Reply:
done.